src/middlewared/middlewared/plugins/pool_/dataset.py

```diff
@@ -563,6 +563,11 @@ async def do_create(self, app, data):
                 'flags': {'BASIC': 'INHERIT'},
                 'type': 'ALLOW'
             })
+        elif data['share_type'] == 'MULTIPROTOCOL':
+            data['casesensitivity'] = 'SENSITIVE'
```

Comment:
Just curious on what happens when user explicitly specifies these attrs to something else ? Do we have validation in place to prevent them from doing that or are these defaults which we want to apply and if user overrides, he/she can happily do it ?

Reply:
It currently overrides inherited or manually specified parameters. This is consistent with other `share_type` entries that we have. We could redesign this field for DragonFish or Electic Eel if needed.

Reply:
@anodos325 we should have validation in place to not allow users to specify these attrs then as this creates a false impression to the user where he assumes he has set something which is not being actually set